tests/output-pcap-log/test.yaml

```diff
@@ -1,4 +1,7 @@
+requires:
+  min-version: 7
+
 checks:
   - file-compare:
-      filename: log.pcap.1444144603
+      filename: log.pcap.0
```

Comment:
I do find this concerning: https://github.com/OISF/suricata-verify/pull/823#pullrequestreview-989510881

Reply:
ah forgot to add the test copy I created

Reply:
resolved with #838